modules/uploading-training-data-to-trustyai.adoc

```diff
@@ -0,0 +1,58 @@
+:_module-type: PROCEDURE
+
+[id="uploading-training-data-to-trustyai_{context}"]
+= Uploading training data to TrustyAI
+
+[role='_abstract']
+Upload training data to use with TrustyAI for bias monitoring or data drift detection.
+
+.Prerequisites
+
+* Your OpenShift cluster administrator added you as a user to the {openshift-platform} cluster and has installed the TrustyAI service for the data science project that contains the deployed models.
+
+* You have training data to upload.
+
+ifndef::upstream[]
+* You authenticated the TrustyAI service as described in link:{rhoaidocshome}{default-format-url}/monitoring_data_science_models/setting-up-trustyai-for-your-project_monitor#authenticating-trustyai-service_monitor[Authenticating the TrustyAI service]. 
+endif::[]
+ifdef::upstream[]
+* You authenticated the TrustyAI service as described in link:{odhdocshome}/monitoring-data-science-models/#authenticating-trustyai-service_monitor[Authenticating the TrustyAI service].
+endif::[]
+
+
+.Procedure
+
+. Set the `TRUSTY_ROUTE` variable to the external route for the TrustyAI service in your project:
++
+----
+TRUSTY_ROUTE=https://$(oc get route/trustyai-service --template={{.spec.host}})
+----
+
+. Send the training data to the `/data/upload` endpoint:
++
+----
+curl -sk $TRUSTY_ROUTE/data/upload  \
+  --header 'Authorization: Bearer ${TOKEN}' \
+  --header 'Content-Type: application/json' \
+  -d @data/training_data.json
+----
+
+. The message `1000 datapoints successfully added to gaussian-credit-model data` appears if the upload was successful.
+
+.Verification
+
+. Verify that TrustyAI has received the data via the `/info` endpoint by inputting this query:
```

Comment:
```suggestion
* Verify that TrustyAI has received the data via the `/info` endpoint by inputting this query:
```

Reply:
Since Step 2 isn't really a step a customer would need to take, I think this can just be a bullet per the guidance here for single-step procedures: https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs